### PR TITLE
Fix html encoding formcreator

### PR DESCRIPTION
**Pull request by btry**
Handle ITIL objects created by Formcreator for GLPI 9.5 or older.

In some cases, the HTML may contain raw BR tag. This PR escapes it.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: src/Console/Database/FixHtmlEncodingCommand.php [hunk 1]
```diff
@@ -325,6 +327,26 @@ private function fixQuoteEntityWithoutSemicolon(string $input): string
...
+     * Those items were generated with GLPI 9.5's flavor of HTML escaping.
+     *
+     * @param string $input
+     * @return string
+     */
+    private function fixUnescapedLineBreak(string $input): string
+    {
+        $output = $input;
+
+        // Add the missing semicolon to &quot; HTML entity
```
Comment (cedric-anne) [inline src/Console/Database/FixHtmlEncodingCommand.php:342]:
```suggestion
```

**Review by cedric-anne — APPROVED** [review]

**Comment by cedric-anne** [comment]
You should rebase, command has been moded to `Console/Diagnostic` namespace.

**Comment by trasher** [comment]
As discussed, this will be postponed

**Comment by falco2001** [comment]
I don’t have the command in the list: glpi:plugins:formcreator:clean_tickets as in the 10.0.5 thanks

**Pull request closed without merging**